sb.web.app/src/locales/en.json

```diff
@@ -173,12 +173,13 @@
         "item_type": "Item Type",
         "choose_file": "Choose file",
         "item_set_main_image": "Set as main image",
-        "until_when": "Until when will this ",
+        "until_when": "When do you need this item/service / until when will this item/service be available",
         "be_available": " be available?",
         "indefinitely": "Indefinitely",
         "location": "Location",
         "price_prompt": "What is the price of the ",
         "new_articleCAPS": "NEW ARTICLE",
+        "edit_articleCAPS": "EDIT ARTICLE",
```

Comment:
really we should do this with css - its just  a matter of adding text-transform: uppercase to the relevant class for the titles and then removing "CAPS" as a  variation thing from en.json